src/SourceBuild/tarball/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/format/solution/Test.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace FormatTest
```

Comment:
What do you think about moving this to the baselines folder?  I would be ok with moving the unformatted file there as well.  If moved, they should be named that more closely associates them to the owning tests.

Thinking about this more maybe we should create a separate folder for `assets` where the unformatted class would go and the baselines folder is a subset.  Then the csproj would only have one folder for all `content`